### ISSUE TITLE
Suggestion: Different tray icons for devices

### ISSUE THREAD

**Issue by cspix** [josetr/AudioDeviceSwitcher#4]
I think it would be nice to be able to use different tray icons for devices. This way you could see at a glance which device is currently selected without having to toggle them to see.

**Comment by SaintPeter** [comment]
Yes, and/or also show the currently selected device when you hover over the icon. Alternatively, change the color of the icon or have user configurable colors on each device.
